workers/unity/Assets/Tests/Performance/PlaymodeTests/ThisShouldRun.cs

```diff
@@ -0,0 +1,75 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+using UnityEngine;
+
+namespace Improbable.Gdk.PlaymodePerformanceTests
+{
+    public class ThisShouldRun : MockBase
```

Comment:
Same question for this one as the `ThisShouldRun` from above, is there any difference between the two or could we combine them so there's only one?

Reply:
They're identical, but this one's for playmode and the other for editmode